Installer_Linux.sh

```diff
@@ -78,6 +78,14 @@ if [ ! -d /data/hdd/misc ]; then
     sudo chown -R ${USER:=$(/usr/bin/id -run)}:$USER /data/hdd/misc
 fi
 
+echo
+echo "Acquiring shape files..."
+if [ ! -d /data/hdd/misc/shapes]; then
+    wget http://navigator.oceansdata.ca/cdn/shapes.tar.bz2
+    sudo tar -xjC /data/hdd/misc -f shapes.tar.bz2
+    sudo rm shapes.tar.bz2
```

Comment:
I don't think the `sudo` prefixes are needed here, since on line 78 above we recursively change the ownership of `/data/hdd/misc`, so naturally when you unzip `shapes.tar.bz2` (without sudo), it'll be the current user who owns this folder.